src/Sarif/Visitors/SarifCurrentToVersionOneVisitor.cs

```diff
@@ -364,7 +364,7 @@ internal NotificationVersionOne CreateNotificationVersionOne(Notification v2Noti
                     Id = v2Notification.Descriptor?.Id,
                     Level = Utilities.CreateNotificationLevelVersionOne(v2Notification.Level),
                     Message = v2Notification.Message?.Text,
-                    PhysicalLocation = CreatePhysicalLocationVersionOne(v2Notification.PhysicalLocation),
+                    PhysicalLocation = CreatePhysicalLocationVersionOne(v2Notification.Locations?[0].PhysicalLocation),
```

Comment:
>. [](start = 86, length = 1)

Missing null check: s/be:
```
v2Notification?.Locations?[0].PhysicalLocation
```

Reply:
i left it be because of LOC 359?

---
In reply to: [278290353](https://github.com/Microsoft/sarif-sdk/pull/1456#discussion_r278290353) [](ancestors = 278290353)

Reply:
oh sorry, right.

---
In reply to: [278291270](https://github.com/Microsoft/sarif-sdk/pull/1456#discussion_r278291270) [](ancestors = 278291270,278290353)